.circleci/config.yml

```diff
@@ -109,7 +116,7 @@ parameters:
   skip-maint:
     type: string
     default: ""
-  # target also used by ma-cf-deploy
+  # target also used by ma-cf-deploy, ma-backstpo
```

Comment:
@weitzman Hope I'm not being too nit-picky, but ma-backstop is spelling is wrong here

Reply:
No problem. Fixed. FYI, the suggestions feature of Github is great for review comments like this. See https://thenextweb.com/dd/2018/10/16/github-launches-suggested-changes-to-make-it-easier-to-collaborate-within-pull-requests/